RavenDB.Identity/UserStore.cs

```diff
@@ -832,7 +834,7 @@ public static void MigrateToV6(IDocumentStore docStore)
                 .Operations
                 .Send(new DeleteByQueryOperation(new Client.Documents.Queries.IndexQuery
                 {
-                    Query = "from IdentityUserByUserNames"
+                    Query = "from {collectionName}"
```

Comment:
Did you mean to do string interpolation here?

Reply:
You are right. Fixed.